### PR TITLE
add lob query test & add nullable test & add batch ttl test

### PR DESCRIPTION
**Pull request by WeiXinChan**
## Summary

- add lob query test & add nullable test & add batch ttl test

### CODE REVIEW

File: test/login/login_test.go [hunk 1]
```diff
@@ -34,7 +35,7 @@ const (
 )
 
 const (
-	passLoginTest = false
+	passLoginTest = true
```
Comment (IHEII) [inline test/login/login_test.go:38]: Should we do this manually?

Comment (WeiXinChan) [reply test/login/login_test.go:38]: yes, it is a special test

Comment (IHEII) [reply test/login/login_test.go:38]: LGTM

**Comment by IHEII** [comment]
LGTM

**Pull request merged**